src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/ModalBox.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import React from 'react';
+import keydown from 'react-keydown';
+import Icon from '../Icon';
+import type {ModalProps} from './types';
+import Actions from './Actions';
+import modalBoxStyles from './modalBox.scss';
+
+type Props = ModalProps & {
+    onRequestClose: () => void,
+}
+
+const CLOSE_ICON = 'times';
+
+export default class ModalBox extends React.PureComponent<Props> {
+    static defaultProps = {
+        actions: [],
+    };
+
+    @keydown('esc') requestClose() {
```

Comment:
rename to `handleRequestClose`?

Reply:
The component doesn't handle a request close call, but really requests to be closed.